### ISSUE TITLE
Unable to login

### ISSUE THREAD

**Issue by sburkeen** [kuovonne/airtableBlock-postToWordPress#3]
Hi, I've installed the JWT Authentication for WP REST API and followed all those directions, including installing WP Rest, altering the .htaccess and wp-config.php file. I'm still getting unable to login. I've double checked, my login is correct. Any suggestions?

**Comment by kuovonne** [comment]
Sometimes where you place the edits in the .htaccess and wp-config.php files matters.

For the .htaccess edits, you need to put the new lines after the comment:  # END WordPress

For the wp-config.php edits, you should put them with the other constant definitions: define(

Also, make sure that you activate the JWT Authentiation plugin in the WordPress dashboard.

**Comment by sburkeen** [comment]
Thanks for responding so quickly. I replaced the keys in wp-config in the correct place in define.

I didn't put the .htaccess edits after end Wordpress. Just changed that and I'm still getting the error.

Do I need to add everything after "CONFIGURATE THE SECRET KEY" like "CONFIGURATE CORS SUPPORT" etc from the JWT plugin?

Really appreciate your help!

**Comment by kuovonne** [comment]
Yes, you also need to configure CORS support and activate the plugin.

**Comment by sburkeen** [comment]
Okay, thank you. I activated that, deactivated the plugin and activated it again, confirmed that all the additions to the htaccess file and wp-config file were there. Tried again, same error. Then I search and found special characters in user passwords sometimes mess with JWT, so I reset that password using only normal characters still didn't work.

I can pull up the JWT plugin here: https://blogs.wnpt.org/arts/wp-json/

I just can't get the token page to pull up. I get this error:  

{"code":"rest_no_route","message":"No route was found matching the URL and request method","data":{"status":404}}

I have php 7.4 installed for the Wordpress site. Pretty much checked everything I could think of. 

I appreciate the help. If you happen to think of something that would fix this give me a shout.
